### PR TITLE
Got Eventual to build and run on Haiku

### PR DESCRIPTION
**Pull request by emily0618**
- Create build.sh that builds/installs shared libraries and apps
- Modify makefiles to link the libraries from a standard directory
- Update the screenshot in README.md
- Remove obsolete files

### CODE REVIEW

**Comment by scottmc** [comment]
Try writing a haikuporter recipe for this and verify that it builds into a .hpkg.  You may find you need to adjust the build script to work in a recipe and standalone.

File: src/EventEditor/Makefile [hunk 1]
```diff
@@ -84,7 +84,7 @@ LIBS= 	TimeRepresentation	\
 #	be recursive, so include all of the paths where libraries can
 #	be found.  Directories where source files are found are
 #	automatically included.
-LIBPATHS= ../../lib
+LIBPATHS= /boot/home/config/non-packaged/lib
```
Comment (owenca) [inline src/EventEditor/Makefile]:
```suggestion
LIBPATHS= ../lib
```

File: src/EventServer/Makefile [hunk 2]
```diff
@@ -80,7 +80,7 @@ LIBS= 	Utilities			\
 #	be recursive, so include all of the paths where libraries can
 #	be found.  Directories where source files are found are
 #	automatically included.
-LIBPATHS= ../../lib
+LIBPATHS= /boot/home/config/non-packaged/lib
```
Comment (owenca) [inline src/EventServer/Makefile]:
```suggestion
LIBPATHS= ../lib
```

File: src/PreferencesPreflet/Makefile [hunk 3]
```diff
@@ -70,7 +70,7 @@ LIBS= 	be		\
 #	be recursive, so include all of the paths where libraries can
 #	be found.  Directories where source files are found are
 #	automatically included.
-LIBPATHS= ../../lib
+LIBPATHS= /boot/home/config/non-packaged/lib
```
Comment (owenca) [inline src/PreferencesPreflet/Makefile]:
```suggestion
LIBPATHS= ../lib
```

**Review by owenca — CHANGES_REQUESTED** [review]
@emily0618 your way works perfectly! The alternative I suggested here is just my preference.

**Comment by emily0618** [comment]
@scottmc I will update the recipe as soon as this pull request is merged.

**Comment by emily0618** [comment]
Updated the [recipe](https://github.com/haikuports/haikuports/pull/4635).

**Pull request merged**